index.html

```diff
@@ -0,0 +1,101 @@
+            <div class="mainTitle">
+                <h3 class="hello">"Hello! Mental Athletes"</h3>
+                <h1>International Association of Memory Global Summit 2022</h1>
+            </div>
+            <p class="place">A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from over 50 countries is taking place in December, in Tashkent, Uzbekistan.</p>
+            <p class="date">2022.12.12(MON) ~ 13(TUE)</p>
+            <p class="venue">@HYATT Regency in Tashkent</p>
+        </section>
+        <section id="main" class="mainProgram">
+            <h3 class="h3_main">Main Program</h3>
+            <hr class="hr_main">
+            <div class="grid-container">
+                <div class="prog">
+                    <img src="./images/conference.png" alt="">
+                    <h4>Lecture</h4>
+                    <p>Listen to the mental athletes from various countries about the great mnemonic techniques.</p>
+                </div>
+                <div class="prog">
+                    <img src="./images/chat.png" alt="">
+                    <h4>Forum</h4>
+                    <p>Have the time to share your thoughts and opinions with experts in mental sports.</p>
+                </div>
+                <div class="prog">
+                    <img src="./images/museum.png" alt="">
+                    <h4>Exhibition</h4>
+                    <p>See and learn about the tools and devices the mental athletes use in the championships.</p>
+                </div>
+                <div class="prog">
+                    <img src="./images/teacher.png" alt="">
+                    <h4>Masterclass</h4>
+                    <p>Learn basic mnemonic techniques from memory experts on memorizing various information.</p>
+                </div>
+                <div class="prog">
+                    <img src="./images/ignite.svg" alt="">
+                    <h4>IAM Ignite</h4>
+                    <p>Get opportunities to network with IAM and WMC competitors around the world, also after the summit.</p>
+                </div>
+            </div>
+            <button type="submit" class="join">Join IAM Global Summit 2022</button>
+            <a class="seeMore" href="#main">See the whole program</a>
+        </section>
+        <section class="speech"></section>
+        <section class="partner">
+            <h3 class="h3_partner">Partner</h3>
+            <hr class="hr_partner">
+            <ul>
+                <li><img src="./images/mozilla.svg" alt="" class="brandLogo"></li>
+                <li><img src="./images/google.svg" alt="" class="brandLogo"></li>
+                <li><img src="./images/airbnb-ar21.svg" alt="" class="brandLogo"></li>
+                <li><img src="./images/Yandex.svg" alt="" class="brandLogo"></li>
+                <li><img src="./images/Hilton.png" alt="" class="brandLogo" id="hilton"></li>
+            </ul>  
```

Comment:
- [ ] All your  `img` tags should contain an alts value so that your web page can have a good accessibility.